### DISCUSSION TITLE
Help with arbitrary background image

### DISCUSSION THREAD

**Discussion by elabdelaziz** [tailwindlabs/tailwindcss#6617] [Help] [answered]
Hello There,
I'm getting an error when I load an image from public folder or src folder using an arbitrary url value
`bg-[url('../public/assets/home/desktop/image-speaker-zx7.jpg')]`
I've tried moving some images to src file and it's not working either.
REPO: [https://github.com/elabdelaziz/Audiophile-Ecom](url)
![tailwind-issue](https://user-images.githubusercontent.com/64102237/146658637-e6401a46-4e46-4702-a801-817fbf80b0f9.png)

**Comment by NovusOpus** [comment, marked as answer]
This should be working:
```html
<div class="bg-[url('/img/logo.svg')] bg-no-repeat">
...
</div>
```

Also you can set the background image with a Style Attribute: 

```html
<div class="bg-no-repeat" style="background-image: url('/img/logo.svg')">
...
</div>
```

Working Example: [Tailwind Play](https://play.tailwindcss.com/6HFQYJfjL1)

**Reply by jacobedawson** [reply]
Setting the background with a style attribute is working, but I can't seem to get it to work with arbitrary values using the bg-[PATH] method for some reason.

**Reply by module17** [reply]
Problem still exists at v3.2.7.  `bg-[url(/path/image.png)]` fails to load an image, but `style="background-image: url(...)"` is just fine on the same element.

**Reply by JohnnyWalkerDigital** [reply]
Having the same issue with 3.3.5

**Reply by Lanix2** [reply]
I having the same issue with 3.4.3

**Comment by coycoylaniba** [comment]
i'm having the same issue `"tailwindcss": "^3.3.2"`

**Comment by pzgz** [comment]
Having same issue, with tailwindcss@3.3.3, looks like this feature is not very stable. 

* Single quote cannot be used, without single quote, it's working in local
* But no matter with or without single quote, it's not working after build into dist and deploy to production
* As solution provided by @NovusOpus , setting URL in style is working

**Reply by adamwathan** [reply]
If you share a reproduction it will be easier to help 👍  My suspicion is that the image isn't actually deployed at the exact URL being used in the class name but it really depends on the project. Often assets like images have their paths rewritten when building for production to include some sort of hash or versioning in the file name, which could cause the path mismatch.
